Public/Src/Sandbox/MacOs/Interop/Interop.xcodeproj/xcshareddata/xcschemes/CoreDumpTester.xcscheme

```diff
@@ -82,10 +82,10 @@
       </BuildableProductRunnable>
    </ProfileAction>
    <AnalyzeAction
-      buildConfiguration = "Debug">
+      buildConfiguration = "release">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Release"
+      buildConfiguration = "release"
```

Comment:
just curious: are any of these lower vs upper casing changes significant?

Reply:
I guess when you were introducing the xcconfig files for the debug / release configuration you changed configuration names maybe? I didn't look at the history - but Xcode takes the name of the configuration which is lowercase in both cases and had a reference to uppercase, resulting in three entries to chose from although only two were present. Also the release xcconfig was the default for debug builds, which made the builds "release" flavored and bxl complain about flavor mismatch.